_projects/integrate-pyflakes-AST.md

```diff
@@ -15,8 +15,10 @@ mentors:
   - adhikasp
 name: "Integrate pyflakes-enhanced AST into coala "
 requirements:
-  - "The participant has a patch to any bear merged. Check out [existing bear proposal](https://github.com/coala/coala-bears/issues)."
-  - "The participant has a patch to pyflakes merged. Check out [easy task](https://bugs.launchpad.net/pyflakes/+bugs?field.tag=easy)."
+  - "The participant has a patch to any bear merged. Check out 
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.

(also applies to next bullet item)